softlayer/disk/softlayer_disk_test.go

```diff
@@ -2,26 +2,36 @@ package disk_test
 
 import (
 	boshlog "github.com/cloudfoundry/bosh-agent/logger"
+	common "github.com/maximilien/bosh-softlayer-cpi/common"
+	fakeclient "github.com/maximilien/softlayer-go/client/fakes"
 
 	. "github.com/maximilien/bosh-softlayer-cpi/softlayer/disk"
 	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 )
 
-var _ = XDescribe("SoftLayerDisk", func() {
+var _ = Describe("SoftLayerDisk", func() {
 	var (
+		fc   *fakeclient.FakeSoftLayerClient
 		disk SoftLayerDisk
 	)
 
 	BeforeEach(func() {
+		fc = fakeclient.NewFakeSoftLayerClient("fake-user", "fake-key")
 		logger := boshlog.NewLogger(boshlog.LevelNone)
-		disk = NewSoftLayerDisk(1234, logger)
+		disk = NewSoftLayerDisk(1234, fc, logger)
 	})
 
 	Describe("Delete", func() {
```

Comment:
I usually use this convention when describing a method: `Describe("#Delete", ...)`